transport/grpc/inbound.go

```diff
@@ -101,6 +102,12 @@ func (i *Inbound) start() error {
 	)
 
 	go func() {
+		if i.t.options.logger != nil {
+			i.t.options.logger.Info("grpc inbound started", zap.String("address", i.listener.Addr().String()))
```

Comment:
Log messages should be full sentences ending with a period.

    "GPRC Inbound started."

Also, FYI, you can avoid paying the cost of `String()` if not needed (if the
log level is Warn, for example) by using `zap.Stringer(i.listener.Addr())`.

Reply:
nit: Can we copy the logger over to the transport rather than accessing it through the transport?

Reply:
Also, it can default to `zap.NewNop` rather than checking `!= nil` before
logging. We pay no extra cost for the above statement with a no-op logger if
we use `zap.Stringer`.

Reply:
We do not do this in fx modules for example - log messages are lowercase with no period.

Example:

```
2017-08-31T16:38:26.829-0700	INFO	systemportfx.git/systemportfx.go:111	started HTTP server on system port	{"component": "systemportfx", "port": "[::]:56925"}
```

My personal opinion is that we keep them lowercase, as other code is doing this, and we don't have a consistent pattern.

Reply:
We also don't copy the logger over in other places, and there's no reason to - we pass the transport through for other things, adding another field is just duplication, ie i dont want both `i.t.logger` and `i.logger` when they are the same thing.

Reply:
RE: punctuation in log messages: Talked offline. Turns out we decided not to do that at some point. I still think capitalization of initialisms should be correct.

    started HTTP inbound
    started GRPC inbound

(I'm not 100% whether it's gRPC, grpc, or GRPC but the logo on grpc.io is all-caps. I don't care either way.)

RE: copying the logger over: I would rather read `i.t.logger` or `i.logger`
than `i.t.options.logger`. That's too many levels and easy to break.

Reply:
It's not easy to break, and it's consistent with the rest of the code - options is guaranteed to not be nil. I rather keep codebases 1. consistent (the rest of the transport/grpc package does this) 2. correct (not copying fields for subjective readability)

Reply:
I don't care strongly enough about the `options` business/we can change it later.